application/controllers/items.py

```diff
@@ -181,15 +179,14 @@ def put(self, item_id=None):
             if 'charity_score' in request.json:
                 item.charity_score = request.json['charity_score']
             if 'charity_score_image' in request.json:
-                item.charity_score = request.json['charity_score_image']
+                item.charity_score_image = request.json['charity_score_image']
```

Comment:
This is the correction I made - looks like we made a copy/paste mistake earlier this morning.

Reply:
oh man -- good catch